src/interfaces/records/handlers/records-query.ts

```diff
@@ -1,19 +1,19 @@
+import type { DidResolver } from '../../../index.js';
 import type { MethodHandler } from '../../types.js';
-import type { DataStore, DidResolver, MessageStore } from '../../../index.js';
 import type { RecordsQueryMessage, RecordsQueryReplyEntry, RecordsWriteMessage } from '../types.js';
 
 import { authenticate } from '../../../core/auth.js';
 import { lexicographicalCompare } from '../../../utils/string.js';
 import { MessageReply } from '../../../core/message-reply.js';
 import type { RecordsWriteMessageWithOptionalEncodedData } from '../../../store/storage-controller.js';
-import { StorageController } from '../../../store/storage-controller.js';
+import type { StorageController } from '../../../store/storage-controller.js';
```

Comment:
i think this should be moved above with all the other `import type`

Reply:
Addressed in https://github.com/TBD54566975/dwn-sdk-js/pull/338/commits/42f36fb0385704079ff7f0a11a32f953d80a25c7